fandogh_cli/workspace.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, workspace_config=None):
         workspace_config = workspace_config or {}
         self.path = workspace_config.get('path', os.getcwd())
         self.zip_file_name = os.path.join(self.path, 'workspace.zip')
-        files = os.listdir(self.path)
+        files = [file.lower() for file in os.listdir(self.path)]
```

Comment:
Hi @733amir, 
Wonder why we need to lower case all file names here?

Reply:
Hi @psycho-ir,
To get case insensitive search of `dockerfile` and other files and directories in the future.

Reply:
what about : 
```
# at the top
import sys
if sys.version[0] == '2':
 from itertools import imap as map

# in here
self.has_docker_file = 'dockerfile' in map(str.lower, files)
```

Reply:
Good! It is more readable now.
What about `.dockerignore` file? shouldn't we consider that too?

```python
files = map(str.lower, os.listdir(self.path))
self.has_docker_ignore = '.dockerignore' in files
self.has_docker_file = 'dockerfile' in files
```

Reply:
I think we should limit the scope of this PR to Dockerfile case, we can work on .dockerignore in another PR.